README.md

```diff
@@ -18,7 +18,53 @@ In fact, to run tests or basically anything with Django management commands, you
 
 ### Environmental Settings
 
-A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make tests` outside of a Docker container, for example.
+A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make test` outside of a Docker container, for example.
```

Comment:
You can run it outside, but yes in some cases you end up with permission issues. I'm working on finding a way to actually solve this, because you should be able to do this stuff outside of the container in a dev environment. The proposed way would be to match the container user's UID with the host user's UID. Unfortunately getting that right with docker is not as easy as it sounds :p

Reply:
Yep, docker can be a pain on these things (and so many other things...) - although to be fair vagrant also has its share of weird behavior when accessing the filesystem from outside the VM. Sounds good for here.